tgui/packages/tgui/interfaces/ShuttleBlueprints.tsx

```diff
@@ -198,27 +197,29 @@ const ShuttleConstruction = (props: ShuttleConstructionProps) => {
   );
 };
 
-const ShuttleConfiguration = (props: ShuttleConfigurationProps) => {
+const ShuttleConfiguration = () => {
   const [name, setName] = useState<string>();
```

Comment:
wild... not your fault, sure, but this should instantiate name as a string and allow ts to infer it, rather than casting it without an initial state (thus making the type wrong, it's then `string | undefined`)

please fix this error while you're here,

```suggestion
  const [name, setName] = useState('');
```